### PR TITLE
[KO] Add missed section description in function.ipynb

### PR DESCRIPTION
**Pull request by the-jb**
'Advanced example: An in-graph training loop' section is missed with the translation. Add translation for section description

### CODE REVIEW

**Comment by the-jb** [comment]
@googlebot I signed it!

File: site/ko/guide/function.ipynb [hunk 1]
```diff
@@ -549,6 +549,20 @@
...
+      "metadata": {
+        "colab_type": "text",
+        "id": "NfpIQUv28Ht4"
+      },
+      "source": [
+        "## 고급 예제: 그래프내 훈련 루프\n",
+        "\n",
+        "이전 섹션은 케라스 레이어나 모델 내부에서 오토그래프를 활용할 수 있는 것을 보여주었습니다. 오토그래프 코드 안에서 케라스 모델을 활용할 수도 있습니다.\n",
+        "\n",
+        "이 예제는 간단한 케라스 모델을 MNIST 에서 어떻게 훈련시키는지를 그래프 내에서 수행되는 전체 훈련 과정-배치 불러오기, 그라디언트 계산, 매개변수 갱신, 검증 정확도, 수렴까지 반복-을 통해 보여줍니다."
```
Comment (jaketae) [inline site/ko/guide/function.ipynb]:
```suggestion
        "이 예제는 배치 불러오기, 그래디언트 계산, 매개변수 갱신, 검증 정확도 계산, 수렴까지 반복 등 그래프 내에서 수행되는 전체 훈련 과정을 통해 간단한 케라스 모델이 어떻게 MNIST 데이터셋에 훈련되는지 보여줍니다."
```

File: site/ko/guide/function.ipynb [hunk 2]
```diff
@@ -549,6 +549,20 @@
...
       "outputs": []
     },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "NfpIQUv28Ht4"
+      },
+      "source": [
+        "## 고급 예제: 그래프내 훈련 루프\n",
```
Comment (jaketae) [inline site/ko/guide/function.ipynb]:
```suggestion
        "## 고급 예제: 그래프 내 훈련 루프\n",
```

**Comment by the-jb** [comment]
@jaketae Thank you for suggestion. I update commits about your suggestion.

**Comment by jaketae** [comment]
@the-jb Thank you for the quick response. Looks good to me, except for the CLA check. Can you make sure that the CLA has been signed? (I see that it has been, but let's double check just to be sure.)

**Comment by lamberta** [comment]
Thanks. Confirmed the CLA.
The bot can get confused if the commit is authored with a different email than the one used to sign the CLA.

**Comment by the-jb** [comment]
@lamberta, @jaketae 
I saw the CLA signing problem occured when adding co-author(@jaketae) on the new commit, so I thought @jaketae needs to sign.
But if it's not, I guess googlebot maybe doesn't recognize when the commit has added new co-author who has already signed the CLA.
Anyway, thank you for the quick fix @lamberta.

**Comment by jaketae** [comment]
@the-jb Good to see that the problem has been fixed! LGTM

**Pull request merged**